cfpb/retirement
# Updates to latest Capital Framework | Remove bower

## Additions

- Adds task to copy icons to static directory.

## Removals

- Removes bower.
- Removes cf enhancements that aren't referenced in the markup.

## Changes

- Updates to Capital Framework 7.x from 1.x.

## Testing

1. Turn on font cdn by setting `@use-font-cdn: true;` on line 29 of `main.less`.
2. Run `gulp build`
3. Run `python manage.py runserver`
4. Visit http://localhost:8000/before-you-claim/ and compare to https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/
  - Enter 1900 for year born to show notification.
5. `npm test` should pass.

## Notes

- `tox` does not pass, but it does not pass on master branch either, so I'm not sure of the issue. I have not touched it, including not updating the icon syntax in the fixtures.
